plugins/woocommerce-blocks/tests/e2e/tests/single-product-template/single-product-template.block_theme.spec.ts

```diff
@@ -41,6 +41,7 @@ const products = [
 ];
 
 for ( const { classes, product, frontendPage } of products ) {
+	// eslint-disable-next-line playwright/valid-describe-callback
 	test.describe( `The Single Product page of the ${ product }`, () =>
```

Comment:
Instead of disabling `playwright/valid-describe-callback`, would it be possible to wrap the describe callback in `{}` so it doesn't return anything (and then, the describe callback is valid)?

Reply:
Nice catch! That describe should definitely not be returning that `test`! 😄

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/46125/commits/d0c5c334b3006218c9117b40540b1bb3690d05c2